docker/docker-compose.yml

```
@@ -61,6 +70,7 @@ services:
       PGRST_DB_SCHEMA: public, storage
       PGRST_DB_ANON_ROLE: anon
       PGRST_JWT_SECRET: ${JWT_SECRET}
+      PGRST_DB_USE_LEGACY_GUC: "false"
```

@steve-chavez - I couldn't see this option in the readme on Dockerhub 
https://hub.docker.com/r/postgrest/postgrest/

Can I confirm that this will automatically get passed through to the config?

---

Yes, but it should be:
```suggestion
      PGRST_DB_USE_LEGACY_GUCS: "false"
```